### PR TITLE
hotfix for issues introduced during merging

### PR DESCRIPTION
**Pull request by JamesMatchett**
Hotfix patches

### CODE REVIEW

**Review by Conormc98 — APPROVED** [review]

**Review by ShaneKainos — APPROVED** [review]
All seems good, pulled and ran the page and all's working fine.

File: Database-Files/dummy.sql [hunk 1]
```diff
@@ -176,9 +214,180 @@ insert into responsibilities (respDesc)
...
+insert into resp_Role(respID, roleID) values (5, 5);
+insert into resp_Role(respID, roleID) values (6, 6);
+insert into resp_Role(respID, roleID) values (7, 7);
+insert into resp_Role(respID, roleID) values (8, 8);
+insert into resp_Role(respID, roleID) values (9, 9);
+insert into resp_Role(respID, roleID) values (1, 10);
+insert into resp_Role(respID, roleID) values (2, 11);
+insert into resp_Role(respID, roleID) values (3, 12);
+insert into resp_Role(respID, roleID) values (4, 13);
+insert into resp_Role(respID, roleID) values (5, 14);
```
Comment (ChrisThompsonK) [inline Database-Files/dummy.sql:393]: Can we get a screenshot of this working

File: ExpressAPI/index.js [hunk 2]
```diff
@@ -43,6 +45,7 @@ app.get('/responsibilities/:bandID', function(req, res) {
     })
 });
 
+//
```
Comment (ChrisThompsonK) [inline ExpressAPI/index.js]: Do we need this

File: kjourney/src/app/app.component.html [hunk 3]
```diff
@@ -13,11 +13,12 @@
       </div>
 
       <div class="full" style="width:100%">
-        <div class="left" style="width:50%; float:left;">
```
Comment (ChrisThompsonK) [inline kjourney/src/app/app.component.html]: I think we need this back in

File: kjourney/src/app/app.component.html [hunk 4]
```diff
@@ -65,16 +66,15 @@
             </div>
           </div>
         </div>
-        <div class="right" style="width:50%; float:right;">
```
Comment (ChrisThompsonK) [inline kjourney/src/app/app.component.html]: This too - something for design meeting

File: Database-Files/dummy.sql (hunk 1 above)
Comment (JamesMatchett) [reply Database-Files/dummy.sql:393]: image

File: ExpressAPI/index.js (hunk 2 above)
Comment (JamesMatchett) [reply ExpressAPI/index.js]: Will take it out

File: kjourney/src/app/_helpers/auth.guard.ts [hunk 5]
```diff
@@ -1,24 +1,39 @@
...
+        } else{
 
         // not logged in so redirect to login page with the return url
         this.router.navigate(['/login'], { queryParams: { returnUrl: state.url } });
         return false;
+        }
+    }
+
+    private checkUserValid():Promise<boolean>{
+        return this.http.get('api/IsUserValid').toPromise().then((res:any) => {
```
Comment (micalexis) [inline kjourney/src/app/_helpers/auth.guard.ts:30]: dont convert to Promise use Observables... keep it consistent!

**Review by ChrisThompsonK — CHANGES_REQUESTED** [review]
Just a couple of small things, happy to approve once changes made and commits squashed.

File: kjourney/src/app/app.component.html (hunk 3 above)
Comment (JamesMatchett) [reply kjourney/src/app/app.component.html]: put back in

File: kjourney/src/app/app.component.html (hunk 4 above)
Comment (JamesMatchett) [reply kjourney/src/app/app.component.html]: take it out or put it back in?

**Review by ChrisThompsonK — APPROVED** [review]
Excellenté

**Comment by JamesMatchett** [comment]
> Excellenté

image

**Review by micalexis — CHANGES_REQUESTED** [review]
minor change required but happy with it 👍

File: kjourney/src/app/_helpers/auth.guard.ts (hunk 5 above)
Comment (JamesMatchett) [reply kjourney/src/app/_helpers/auth.guard.ts:30]: As per our previous conversation, resolving this.

**Review by micalexis — APPROVED** [review]

**Pull request merged**